Mathlib/LinearAlgebra/UnitaryGroup.lean

```diff
@@ -198,6 +198,23 @@ set_option linter.uppercaseLean3 false in
 
 end UnitaryGroup
 
+section specialUnitaryGroup
+
+variable (n) (α)
+
+/--`Matrix.specialUnitaryGroup` is the group of unitary `n` by `n` matrices where the determinant
+is 1. (This definition is only correct if 2 is invertible.)-/
+abbrev specialUnitaryGroup := unitaryGroup n α ⊓ MonoidHom.mker detMonoidHom
+
+variable {n} {α}
+
+theorem mem_specialUnitaryGroup_iff :
+    A ∈ specialUnitaryGroup n α ↔ A * star A = 1 ∧ A.det = 1:= by
+  rw [← mem_unitaryGroup_iff]
+  rfl
```

Comment:
I think this should be definitional. There's no reason to pick a direction here (i.e., `A * star A` vs. `star A * A`)
```suggestion
    A ∈ specialUnitaryGroup n α ↔ A ∈ unitaryGroup n α ∧ A.det = 1 :=
  Iff.rfl
```